### PR TITLE
Add docs page for IPv6-only

### PR DESCRIPTION
**Pull request by bschimke95**
This is quite similar to the dualstack documentation setup.

### CODE REVIEW

**Review by HomayoonAlimohammadi — APPROVED** [review]
Thanks a lot @bschimke95! Great work! LGTM

**Review by louiseschmidtgen — COMMENTED** [review]
Nice work Ben, left a couple of comments!

**Review by eaudetcobello — APPROVED** [review]
Nice work!

File: docs/src/snap/howto/networking/ipv6.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+- Your environment supports IPv6.
+- Network infrastructure, such as routers, firewalls, and DNS, are configured
+to handle IPv6 traffic.
+- Any underlying infrastructure (e.g. cloud providers, bare metal setups)
+must be IPv6-compatible.
+
+## Setting Up an IPv6-Only Cluster
+
+The process of creating an IPv6-only cluster involves specifying only IPv6
+CIDRs for Pods and Services during the bootstrap process. Unlike dual-stack,
```
Comment (louiseschmidtgen) [inline docs/src/snap/howto/networking/ipv6.md:23]:
```suggestion
CIDRs for pods and services during the bootstrap process. Unlike dual-stack,
```

File: docs/src/snap/howto/networking/ipv6.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+Once the cluster is up, verify that all pods are running:
+
+```sh
+sudo k8s kubectl get pods -A
+```
+
+Deploy a pod with an nginx web-server and expose it via a service to verify
+connectivity of the IPv6-only cluster. For that, create a manifest file
+`nginx-ipv6.yaml` with the following content:
```
Comment (louiseschmidtgen) [inline docs/src/snap/howto/networking/ipv6.md:66]:
```suggestion
connectivity of the IPv6-only cluster. Create a manifest file
`nginx-ipv6.yaml` with the following content:
```

File: docs/src/snap/howto/networking/ipv6.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+  ipFamilies:
+  - IPv6
+  ports:
+  - port: 80
+    protocol: TCP
+  selector:
+    run: nginx-ipv6
+```
+
+Deploy it with:
```
Comment (louiseschmidtgen) [inline docs/src/snap/howto/networking/ipv6.md:106]:
```suggestion
Deploy the web-server and its service by running:
```

**Review by louiseschmidtgen — APPROVED** [review]
A few small nits, then we are good to merge. Thanks Ben!

**Pull request merged**